apps/tools/update_config.sh

```diff
@@ -30,40 +30,49 @@ function write_config(){
 	sed -i "s/CONFIG_USER_ENTRYPOINT.*/CONFIG_USER_ENTRYPOINT=$UPDATED_ENTRYPOINT/" $config_path
 
 	CONFIG_ENTRY_LIST=`sed -n '/^CONFIG_ENTRY_/p' $config_path`
-	if [ "$CONFIG_ENTRY_LIST" != "CONFIG_ENTRY_MANUAL" ]; then
+	if [ "$CONFIG_ENTRY_LIST" != "CONFIG_ENTRY_MANUAL" ]
```

Comment:
DON'T mix different kinds of changes.
Please revert formatting.
This makes difficult to check what are changed.